bigchaindb/tendermint/lib.py

```diff
@@ -39,12 +40,55 @@
              'broadcast_tx_commit')
 
 
-class BigchainDB(Bigchain):
+class BigchainDB(object):
+    """Bigchain API
+
+    Create, read, sign, write transactions to the database
+    """
+
+    BLOCK_INVALID = 'invalid'
+    """return if a block has been voted invalid"""
+
+    BLOCK_VALID = TX_VALID = 'valid'
+    """return if a block is valid, or tx is in valid block"""
+
+    BLOCK_UNDECIDED = TX_UNDECIDED = 'undecided'
+    """return if block is undecided, or tx is in undecided block"""
+
+    TX_IN_BACKLOG = 'backlog'
```

Comment:
The parameters `BLOCK_INVALID`, `BLOCK_VALID`, `BLOCK_UNDECIDED` , `TX_IN_BACKLOG` are not required anymore and should be removed.

Reply:
It breaks a bunch of tests when I remove those parameters. I suggest we excavate them in a separate PR.

Reply:
I think this is a good idea. This PR is already big enough.

Reply:
I'm taking care of removing the parameters in a different PR